### ISSUE TITLE
Add an accessibility section to the documentation for each component

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lux-design-system#337]
### Introduction

For each component, we should have an accessibility section that describes:

- accessibility features of the component
- considerations that you need to keep in mind when using this component in your application

We are particularly inspired by [Mercury, the design system from the Canada Post](https://design.canadapost-postescanada.ca/en/mercury/components/breadcrumbs.page#!navtabd230042e25), which has an Accessibility Tab in the documentation for each of its components.

### Acceptance criteria

- [ ] Each component has an accessibility section

### Implementation notes

We could create a [custom component for displaying Accessibility information](https://vue-styleguidist.github.io/docs/Customize.html) in the documentation.
